cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -605,32 +605,25 @@ Which one you use will depend on your flow:
 Regardless of which of the two methods you use, if the call succeeds, it will return your issued credentials.
 These can then be used in other Wallet-SDK APIs or [serialized for storage](#verifiable-credentials).
 
-### Issuer Metadata
 
-Depending on your use case, you may want to check the issuer's metadata. To do this, call the `issuerMetadata` method
-which will return an `IssuerMetadata` object. Note that some of the methods available on this object and its nested
-sub-objects, can return null/nil/an empty value if the issuer does not supply the corresponding data in its metadata.
-You should ensure that these situations are handled accordingly. Read this section carefully to see which methods may
-return null/nil/an empty value.
+### Issuer and Credential Preview API
+The issuer metadata provided human-readable issuer and credential details. Use following API to get the
+preview details from the credential offer.
 
-The `IssuerMetadata` object has the following methods on it:
 
-* `credentialIssuer()`: Returns the issuer's identifier.
-* `localizedIssuerDisplays()`: Returns the [LocalizedIssuerDisplays](#LocalizedIssuerDisplays) object, which contains
-  display information for the issuer.
-* `supportedCredentials()`: Returns the [SupportedCredentials](#SupportedCredentials) object, which contains technical
-  and display data about the credential types that this issuer can issue.
+```kotlin
+previewData = Display.resolveCredentialOffer(issuanceInteraction.issuerMetadata(),issuanceInteraction.offeredCredentialsTypes(), "locale" )
```

Comment:
maybe it is good to mention that if an empty string is passed the default locale will be used

Reply:
[Added a note](https://github.com/trustbloc/wallet-sdk/compare/9e049bbb49e8bde30154b0367f32ace11a417f37..9ab52e6dd82d684091b1855b46dd3084ea489692) mentioning en-US will be the default one.